scripts/globals/mobskills/charged_whisker.lua

```diff
@@ -2,21 +2,19 @@
 -- Charged Whisker
 -- Deals Lightning damage to enemies within area of effect.
 ---------------------------------------------------
-
-require("scripts/globals/settings");
-require("scripts/globals/status");
-require("scripts/globals/monstertpmoves");
-
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
+require("scripts/globals/status")
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-    return 0;
-end;
+    return 0
+end
 
 function onMobWeaponSkill(target, mob, skill)
-    local dmgmod = 1;
-    local info = MobMagicalMove(mob,target,skill,mob:getWeaponDmg()*2.8,dsp.magic.ele.THUNDER,dmgmod,TP_MAB_BONUS,1);
-    local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_MAGICAL,MOBPARAM_THUNDER,MOBPARAM_WIPE_SHADOWS);
-    target:delHP(dmg);
-    return dmg;
-end;
+    local dmgmod = 2.5
```

Comment:
> -Charged Whisker (dmgmod)
From researching websites as well as retail testing, this is what I felt was the best number to put in there for more retail realistic play. It's nothing more than a best guess because I did not spend enough time to really get a most accurate number. If we don't like this I can just put it back.

@teschnei I got no idea how hard that should hit or what the actual retail math is we are simulating in these variables (other than "one of them is its weapon and another is its str or int or wtf ever")

you review it :stuck_out_tongue:

Reply:
all of our mobskills are guesses anyways, I don't think it's a problem - he did exactly what we normally do with new mobskills

Reply:
:+1:

Reply:
if someone has a better answer or would like something different, I will change it to whatever it needs to be